src/scripts/audit/tool.sh

```diff
@@ -0,0 +1,305 @@
+#!/bin/bash
+# Copyright 2020, Oracle Corporation and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+
+#set -x
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/utility.sh
+
+function usage {
+  echo "usage: ${script} [-i] [-s] [-w] [-h]"
+  echo "  -i File containing list of images to scan in [Repository]:[Tag]@[Hash] format."
+  echo "  -k File containing Kubernetes Node information in json format (output of 'kubectl get nodes -o json')."
+  echo "  -w Working directory for the generated files. Defaults to './work' dir."
+  echo "  -s Silent mode."
+  echo "  -d Disable Image Validation"
```

Comment:
What kind of image validation is this? Does this mean "Disable image hash validation" or something else?

Reply:
It's a validation check for images that are present in the node information file but have not been pulled on current machine. In other words, it validates if any images are missing on current machine when option to "scan all images" is selected. I have clarified the usage and changed variable name. Please let me know if you have any suggestions for the wordings. Thanks.